hw/top_earlgrey/ibex_pmp_reset_pkg.core

```diff
@@ -0,0 +1,22 @@
+CAPI=2:
+# Copyright lowRISC contributors (OpenTitan project).
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+# XXX: This name is currently required as global identifier until we have
+# support for "interfaces" or a similar concept.
+# Tracked in https://github.com/olofk/fusesoc/issues/235
```

Comment:
I saw the related issue is closed in the upstream?

Reply:
Could it be that our downstream fork doesn't include the fix for this issue?

Reply:
Just discussed with @Razer6; this isn't blocking and can be resolved in a follow-up PR (potentially together with other files that are affected, if this is a copy-paste)